litr/src/main/java/com/linkedin/android/litr/io/CaptureMediaSource.kt

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright 2022 LinkedIn Corporation
+ * All Rights Reserved.
+ *
+ * Licensed under the BSD 2-Clause License (the "License").  See License in the project root for
+ * license information.
+ *
+ * Author: Ian Bird
+ */
+package com.linkedin.android.litr.io
+
+import android.media.MediaCodec
+import android.media.MediaFormat
+import android.util.Log
+import android.view.Surface
+import com.linkedin.android.litr.codec.Decoder
+import com.linkedin.android.litr.codec.Frame
+import com.linkedin.android.litr.codec.PassthroughDecoder
+import com.linkedin.android.litr.exception.MediaSourceException
+import java.nio.ByteBuffer
+
+private const val TAG = "CaptureMediaSource"
+private const val DECODER_NAME = "CaptureMediaSource.Decoder"
+
+private const val MIME_TYPE_RAW = "video/raw"
```

Comment:
You can add this to `MimeType` class

Reply:
Will do. I originally resisted because it's not really a "standard" mime type, but happy to move.